packages/main/src/components/Form/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { Grid } from '@ui5/webcomponents-react/lib/Grid';
+import React, { FC, forwardRef, ReactElement, ReactNode, ReactNodeArray, Ref, useMemo } from 'react';
+import { CommonProps } from '../../interfaces/CommonProps';
+import { Title } from '@ui5/webcomponents-react/lib/Title';
+import { TitleLevel } from '@ui5/webcomponents-react/lib/TitleLevel';
+import styles from './Form.jss';
+import { createUseStyles } from 'react-jss';
+import { useViewportRange } from '../../../../base/src/hooks/useViewportRange';
+import { FormGroup } from './FormGroup';
+import CurrentRange from './CurrentViewportRange';
+import { JSSTheme } from '../../interfaces/JSSTheme';
+
+export interface FormPropTypes extends CommonProps {
+  /**
+   * Components that are placed into Form.
+   */
+  children: ReactNode | ReactNodeArray;
+  /**
+   * Form title
+   */
+  title?: string;
+}
+
+let formGroups, updatedTitle;
```

Comment:
Declaring those outside of the component actually may break if you use two forms on the same screen!